aws/table_aws_wafregional_web_acl.go

```diff
@@ -0,0 +1,295 @@
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWafRegionalWebAcl,
+				Transform:   transform.FromField("WebACLArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listWafRegionalWebAcls(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	maxItems := int32(100)
+	params := &wafregional.ListWebACLsInput{}
+
+	// Reduce the basic request limit down if the user has only requested a small number of rows
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxItems {
+			params.Limit = limit
+		}
+	}
+
+	// Create session
+	svc, err := WAFRegionalClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wafregional_web_acl.listWafRegionalWebAcls", "client_error", err)
+		return nil, err
+	}
+
+	// API doesn't support aws-sdk-go-v2 paginator as of date
+	pagesLeft := true
+	for pagesLeft {
+		response, err := svc.ListWebACLs(ctx, params)
+		if err != nil {
+			plugin.Logger(ctx).Error("aws_wafregional_web_acl.listWafRegionalWebAcls", "api_error", err)
+			return nil, err
+		}
+
+		for _, webAcl := range response.WebACLs {
+			d.StreamListItem(ctx, webAcl)
+
+			// Context may get cancelled due to manual cancellation or if the limit has been reached
+			if d.RowsRemaining(ctx) == 0 {
+				return nil, nil
+			}
+		}
+
+		if response.NextMarker != nil {
+			pagesLeft = true
+			params.NextMarker = response.NextMarker
+		} else {
+			pagesLeft = false
+		}
+	}
+
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getWafRegionalWebAcl(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+
+	var id string
+	if h.Item != nil {
+		data := wafRegionalWebAclData(h.Item, ctx, d, h)
+		id = data["ID"]
+	} else {
+		id = d.EqualsQuals["web_acl_id"].GetStringValue()
+		if id == "" {
+			return nil, nil
+		}
+	}
+
+	// Create Session
+	svc, err := WAFRegionalClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wafregional_web_acl.getWafRegionalWebAcl", "client_error", err)
+		return nil, err
+	}
+
+	params := &wafregional.GetWebACLInput{
+		WebACLId: aws.String(id),
+	}
+
+	op, err := svc.GetWebACL(ctx, params)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wafregional_web_acl.getWafRegionalWebAcl", "api_error", err)
+		return nil, err
+	}
+
+	return op.WebACL, nil
+}
+
+// ListTagsForResource.NextMarker return empty string in API call
+// due to which pagination will not work properly
+// https://github.com/aws/aws-sdk-go/issues/3513
+func listTagsForWafRegionalWebAcl(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	data := wafRegionalWebAclData(h.Item, ctx, d, h)
+
+	// Create session
+	svc, err := WAFRegionalClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wafregional_web_acl.listTagsForWafRegionalWebAcl", "client_error", err)
+		return nil, err
+	}
+
```

Comment:
@karanpopat could you please add the unsupported region check here for the service client?